pkg/fakerp/fakerp.go

```diff
@@ -212,6 +212,8 @@ func enrich(cs *api.OpenShiftManagedCluster) error {
 		Secret:   os.Getenv("AZURE_CLIENT_SECRET"),
 	}
 
+	// /subscriptions/{subscription}/resourcegroups/{resource_group}/providers/Microsoft.ContainerService/openshiftmanagedClusters/{cluster_name}
+	cs.ID = fmt.Sprintf("/subscriptions/%s/resourcegroups/%s/providers/Microsoft.ContainerService/openshiftmanagedClusters/%s", cs.Properties.AzProfile.SubscriptionID, cs.Properties.AzProfile.ResourceGroup, cs.Properties.AzProfile.ResourceGroup)
```

Comment:
Use `cs.Name` instead of `cs.Properties.AzProfile.ResourceGroup` for the cluster name.

Reply:
use resourceid.ResourceID() instead of fmt.Sprintf()